postman/schemas/schemas/Run.yaml

```diff
@@ -0,0 +1,8 @@
+components:
+  schemas:
+    RunConfiguraion:
+      type: object
+      properties: {}
+    RunStatus:
+      type: object
+      properties: {}
```

Comment:
Can't we specify this object further?
Like 
```json
{"progress": "float[0-1]", "sim-running": "bool"}
```

Reply:
I would like to leave it unspecified at the moment like the configuration schemas 😇 I would like to keep the schemas uniformly empty for now 😄